catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -268,6 +278,22 @@ public void setAttributeOverrides(List<String> attributeOverrides) {
         this.attributeOverrides = attributeOverrideMap;
     }
 
+    public void setBlacklistedFilePatterns(List<String> blacklistedFilePatterns) {
```

Comment:
this has security implications as well and should probably use the list of file extensions that we already have: https://github.com/codice/ddf/blob/master/distribution/ddf-common/src/main/resources/etc/system.properties#L118

You can add the thumbnail and tmp files to that list instead.